devicetypes/smartthings/zwave-radiator-thermostat.src/zwave-radiator-thermostat.groovy

```diff
@@ -84,7 +86,7 @@ metadata {
 }
 
 def initialize() {
-	sendEvent(name: "checkInterval", value: 4 * 60 * 60 + 24 * 60 , displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	sendEvent(name: "checkInterval", value: checkInterval , displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
I wonder the value of defining `ping()` for this DTH, since all the devices on it are sleepy. Certainly I don't think `"offlinePingable"` should be set since it's not a life safety device.

Reply:
Not all, Evespring device isn't sleepy.